rir/src/R/Printing.h

```diff
@@ -0,0 +1,17 @@
+#ifndef RIR_R_PRINT
+#define RIR_R_PRINT
+
+#include "R/r.h"
+#include "utils/capture_out.h"
+
+namespace rir {
+
+static std::string dumpSexp(SEXP src, size_t length = 50) {
```

Comment:
Maybe it would be useful to also include the type of the sexp in the dump? But not sure..

Reply:
I mostly use it for the disassembly, and there i mainly care about it not being cluttered. for other cases maybe...